noxfile.py

```diff
@@ -98,15 +98,18 @@
 
 CURRENT_DIRECTORY = pathlib.Path(__file__).parent.absolute()
 
+# Excludes the duplicated tests:
```

Comment:
Owlbot is going to overwrite this. https://github.com/googleapis/synthtool/blob/master/synthtool/gcp/templates/python_library/noxfile.py.j2#L134-L143

We can do a replace rule if needed, but I don't think that's necessary if we remove the generic "kokoro" run.

Reply:
Reversed the changes.